libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/SalesforceListActivity.java

```diff
@@ -26,16 +26,17 @@
  */
 package com.salesforce.androidsdk.ui;
 
-import android.app.ListActivity;
 import android.os.Bundle;
 import android.view.KeyEvent;
 
+import androidx.appcompat.app.AppCompatActivity;
+
 /**
  * Abstract base class for all Salesforce list activities.
  *
  * @author bhariharan
  */
-public abstract class SalesforceListActivity extends ListActivity implements SalesforceActivityInterface {
```

Comment:
We should leave SalesforceListActivity alone since it has been extended by customers and they will expect to be in a ListActivity.
That being said our sample (and template code) should stop using it and instead subclass SalesforceActivity (the AppCompatActivity).

Reply:
Great feedback.  I can pivot to that easily enough.  Should we add a task to forward the deprecation on @ListActivity to our subclass of it here so customers remain informed of the underlying Android SDK deprecation?

Reply:
Take a gander at 👉🏻 https://github.com/forcedotcom/SalesforceMobileSDK-Android/pull/2399/commits/9caed74219b222e9c9b9031d315b622544fbdf53

Reply:
Deprecation on our subclass makes sense.

Reply:
Thanks.  How does the deprecation message look in 👉🏻 https://github.com/forcedotcom/SalesforceMobileSDK-Android/pull/2399/commits/c3b7572c630b3498ce9b2c3dd9b6f4ce2b7b2903 ? I forwarded some of the copy directly from `ListActivity` to provide concrete guidance.

Reply:
👍